posthog/urls.py

```diff
@@ -210,6 +210,7 @@ def opt_slash_path(route: str, view: Callable, name: Optional[str] = None) -> UR
     opt_slash_path("api/user/change_password", user.change_password),
     opt_slash_path("api/user/test_slack_webhook", user.test_slack_webhook),
     opt_slash_path("api/user", user.user),
+    opt_slash_path("api/v2/user", user.UserViewSet.as_view({"get": "retrieve", "patch": "partial_update"})),
```

Comment:
A few points:
1. Let's keep newly added/reworked endpoints plural, i.e. we have `/api/{organizations,plugins,invites,projects}`, so this should be `/users`.
2. Do we have a plan for putting other up-to-date endpoints under `v2`? This will be awkward with no versioning if we want to keep the old endpoint for a longer time, but also awkward with just one `v2` endpoint in the API, so I think we should plan for that.
3. Since this is now DRF 👏  shouldn't the Viewset be registered in `posthog/api/__init__.py` with `router.register(r"users", user.UserViewSet)`? (actually this goes for `Organization*SignupViewSet`s below too, DRF ViewSets should be registered with the DRF router - root one or nesting ones)

Reply:
1. Opened a Slack thread to discuss briefly. Will update outcome here afterwards.
Re 2. Definitely. I was planning we could use [namespace versioning](https://www.django-rest-framework.org/api-guide/versioning/#namespaceversioning), though because the legacy endpoint is not on DRF, it makes more sense to do versioning for this particular endpoint routing-based.
3. 👍 agreed. For the `Organization*SignupViewSet` endpoints, I have a separate PR, will address later.

Reply:
Per conversation on [Slack](https://posthog.slack.com/archives/C0113360FFV/p1617635186123500), we'll do `GET /users/@me/` as the new endpoint for this. This means that as we won't be changing any existing endpoints, we can defer versioning for later. Updating PR now.